.azuredevops/components/ROCmValidationSuite.yml

```diff
@@ -59,10 +63,17 @@ jobs:
       parameters:
         dependencyList: ${{ parameters.rocmDependencies }}
         dependencySource: tag-builds
+# Set link to redirect llvm folder
+  - task: Bash@3
+    displayName: update cmake
```

Comment:
Can this be changed to "create symlink" or similar

Reply:
good catch